examples/conversation_with_agent_with_finetuned_model/conversation_with_agent_with_finetuned_model.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+"""
+This script sets up a conversational agent using
+AgentScope with a Hugging Face model.
+It includes initializing a FinetuneDialogAgent,
+loading and fine-tuning a pre-trained model,
+and conducting a dialogue via a sequential pipeline.
+The conversation continues until the user exits.
+Features include model and tokenizer loading,
+and fine-tuning on the lima dataset with adjustable parameters.
+"""
+# pylint: disable=unused-import
```

Comment:
remove the disable here

Reply:
If I remove it, there will be error 'W0611: Unused HuggingFaceWrapper imported from huggingface_model (unused-import)' when running pre-commit; furthermore, removing `from huggingface_model import HuggingFaceWrapper` will cause the default model wrapper being used and lead to error. Move `HuggingFaceWrapper` to `agentscope/src/agentscope/models` might solve this issue though.

Reply:
Can I proceed to make `HuggingFaceWrapper` part of `agentscope/src/agentscope/models` to resolve this issue?